README.md

```diff
@@ -29,85 +29,7 @@ git clone https://github.com/alphagov/gds-pre-commit.git ~/.gds-pre-commit/
-
-```shell
-[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
-[INFO] Once installed this environment will be reused.
-[INFO] This may take a few minutes...
-[INFO] Installing environment for git@github.com:Yelp/detect-secrets.
-[INFO] Once installed this environment will be reused.
-[INFO] This may take a few minutes...
-Detect Private Key.......................................................Passed
-Detect secrets...........................................................Passed
-```
-
-
-
-# gds-pre-commit
-This repository is here to assist GDS users in setting up pre-commit hooks that can improve the quality and security of projects hosted on GitHub.
-
-## Secrets
-One of the main motivations for using pre-commit hooks is to prevent secrets being pushed to GitHub repositories. When we say secrets we mean things like private keys, API tokens, SSH keys, AWS keys or Slack keys. All of these 'secrets' are used to authenticate or authorise users to services we use or own and would be beneficial for an attacker to steal.
-
-The detect-secrets tool was tested against AWS keys, a private SSH key and a random hash with the tool successfully catching them all, however pre-commit hooks should not be used as a catch-all solution and users should ensure they are following Secure Software Development Lifecycles.
-
-## pre-commit framework
-One of the easiest ways to get started with pre-commit hooks is by using the [pre-commit framework](https://pre-commit.com/).
-
-Read on for step-by-step instruction:
-
-### Installing pre-commit manually
-#### Please note that the quick install section at the top will run these commands globally
-
-To get started you can either run:
-
-`$ brew install pre-commit`
-
-or
-
-`$ pip3 install pre-commit`
-
-## detect-secrets
-The detect-secrets pre-commit hook requires some initialisation before it is run with the framework.
-
-First you will need to install detect-secrets system-wide:
-
-`$ pip3 install detect-secrets`
-
-Then run the following commands in your local repository:
-
-`$ detect-secrets scan > .secrets.baseline`
-
-`$ detect-secrets audit .secrets.baseline`
-
-This will create a baseline for your repository, initialising [plugins](https://github.com/Yelp/detect-secrets/tree/master/detect_secrets/plugins) used and then scan all of the files in your repository. It will ask you about potential secrets it finds and if they are to real secrets or false positives.
-
-The newly created `.secrets.baseline` file should be committed to Github.
-
-
-Once that's completed, you're all done! If you want to add any other hooks, take a look at the [extensive list](https://pre-commit.com/hooks.html). There are various handy linters and formatters for Go, Ruby, Python and Terraform.
-
-## Supported editors
-This pre-commit hook has been tested and is working with the following editors:
-
-* Atom
-* VScode
-* IntelliJ
-* Pycharm
-* Emacs
-* Vim (with Fugitive.vim)
-
+ - [Usage](usage.md)
```

Comment:
I think the Usage should be in the main README, don't we just want to split off the information that's not critical to set up and running the tool? @0atman @danjoneslf

Reply:
That's not what it says here https://trello.com/c/uzKZhQZC/241-gds-pre-commit-readme-remove-content-after-usage-into-a-separate-linked-md-file
(

Reply:
Also, the Usage is WAAAAY to complicated for the normal user, I agree with the decision to split it up.

Reply:
Yes it is...Remove content _after_ "Usage"

Reply:
Two paragraphs with an output snippet for each too complicated for developers?